crates/fluvio-cli/src/error.rs

```diff
@@ -47,6 +47,8 @@ pub enum CliError {
 
```

Comment:
remove other changes

Reply:
Seconded, try to make this pr just be the disable